### ISSUE TITLE
Broken categories

### ISSUE THREAD

**Issue by RubixDev** [Crec0/carpet-rules-database#14]
It seems like some commit between fbb2e3d4f83ca954debd81b4881adbf4745dfb2c and 7e99de8f2cd2b5cd05de44a8c6cd9998ef327614 broke something with the category parsing. Ever since the following db update (92a5d6c7ff8a3d1c2ecc56612301d3e41331f8d5), most category names include multiple categories and commas. For example `railMissingUpdateAfterPushFix` from carpet-fixes used to have the categories

```json
[
  "BUGFIX",
  "CARPETFIXES",
  "EXPERIMENTAL"
]
```

but now has 

```json
[
  "BUGFIX",
  "CARPETFIXES,BUGFIX",
  "CARPETFIXES,BUGFIX,EXPERIMENTAL"
]
```

which is obviously wrong. Many rules also just have all categories in one string now. Take for example `unpackableWool` from Rug: what used to be 

```json
[
  "CRAFTING",
  "RUG",
  "SURVIVAL"
]
```

is now 

```json
[
  "CRAFTING, SURVIVAL, RUG"
]
```

**Comment by Crec0** [comment]
yeah I am aware of this since I introduced it (Was fixing something else and this was the side effect). I plan on redoing the parser again from scratch, will be fixed at that time.
